### DISCUSSION TITLE
is crossing chicane similar to one on this photo? Or to the depicted one? [image in SC changed]

### DISCUSSION THREAD

**Discussion by matkoniecz** [streetcomplete/StreetComplete#3635] [Feedback]
One of users was asking for help due to being confused by chicane barriers and suggested to make them closer to actually encountered ones (skeletal, barrier also on sides
![photo_2022-01-11_18-03-59](https://user-images.githubusercontent.com/899988/149076476-842ca4c8-7ae8-47bd-bb6c-122d39c79568.jpg)
![photo_2022-01-11_18-03-59 (2)](https://user-images.githubusercontent.com/899988/149076482-9a55d94c-617f-4100-9fea-79b5d0c46bb3.jpg)
)
https://wiki.openstreetmap.org/wiki/Key%3Acrossing%3Achicane

file: https://github.com/streetcomplete/StreetComplete/blob/master/res/graphics/railway%20crossing%20icons/chicane.svg
report: https://t.me/OpenStreetMapPolska/7334

**Comment by matkoniecz** [comment]
So the question is: 

is chicane, as depicted on the current image, present somewhere? (no barrier on sides, red-white area, just two supports)

or is chicane as on image more popular? (barriers on sides, more skeletal, multiple attachments to the ground)

**Comment by mnalis** [comment]
I agree it would be better to draw them with sides and closer (or even a little interleaved like [here](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Ftse1.mm.bing.net%2Fth%3Fid%3DOIP.nZvhNOUUTzFb5jTswu1trAHaEK%26pid%3DApi&f=1)) to each other, as in your picture.

There are both types in Croatia, but even simple ones (as currently depicted in SC) are usually placed so there is some other barrier around them, like [this](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Ftse4.mm.bing.net%2Fth%3Fid%3DOIP.rpADp4rvafq4pUAotfF3ZAHaEK%26pid%3DApi&f=1).

So the ones with sides would be more understandable here too if that can be easily depicted. Or alternatively putting three of them more interleaved like in my first picture would also work. But they are definitively too far apart in current SC image.

**Comment by matkoniecz** [comment]
olo81 has drawn CC0 svg based on this one:

![photo_2022-01-12_15-02-53](https://user-images.githubusercontent.com/899988/149154966-da6a3c27-ea65-4036-9396-93d21cd3d458.jpg)
https://gist.github.com/matkoniecz/dd27deb1ad6aefa9345e85465a8c11f8

**Reply by matkoniecz** [reply]
![screen03](https://user-images.githubusercontent.com/899988/149389243-6af48a59-d104-49cc-9b64-efc706942ddc.png)
![screen02](https://user-images.githubusercontent.com/899988/149389249-46655be6-0f50-4fa7-ae22-a8bf0b0ebd82.png)
Partially lost in the night mode but seems clearly superior in my opinion.

I plan on committing it unless someone will protest.

**Comment by smichel17** [comment]
If there are two common types, one idea could be to make the image show one of each, on the near and far side of the tracks.
